docs/endpoints/endpoints.rst

```diff
@@ -0,0 +1,743 @@
+Globus Compute Endpoints
+************************
+
+A Globus Compute Endpoint is a process launched by the user to serve as a conduit for
+executing functions on a computing resource.  The Compute Endpoint process manages the
+site‑specific interactions for executing functions, leaving users with only a single API
+necessary for running code on their laptop, campus cluster, or even leadership‑class HPC
+machines.
+
+In the mental model of Globus Compute, Endpoints are the remote instance:
```

Comment:
```suggestion
In the mental model of Globus Compute, Endpoints runs on the user-accessible remote instance:
```

to more clearly delineate between Compute and user-land?

Reply:
I ... don't think so, not here.  This section is about framing the mental model.  The bullet point below delineates the user-accessible part.